python/lsst/ts/externalscripts/maintel/make_lsstcam_calibrations.py

```diff
@@ -47,6 +51,7 @@ def __init__(self, index=1):
         )
         self._lsstcam = None
         self._ocps_group = None
+        self._mtcs = None
```

Comment:
So, the reason we have `lsstcam` and `ocps_group` being "private" (the leading "\_") is because the `camera`  and `ocps_group` are properties for the abstract class. For the `MTCS` I we don't have such constraint so I think you can drop the "\_".